src/Goerli-DssSpell.t.sol

```diff
@@ -565,4 +560,31 @@ contract DssSpellTest is GoerliDssSpellTestBase {
             8 days
         );
     }
+
+    // NOTE: Only executable by forge
```

Comment:
This can actually be executed in dapptools just fine.

Reply:
Done.